corehq/apps/userreports/tasks.py

```diff
@@ -35,7 +35,6 @@ def rebuild_indicators(indicator_config_id):
             adapter.save(doc)
         except DataError as e:
             logging.exception('problem saving document {} to table. {}'.format(doc['_id'], e))
-    adapter.engine.dispose()
```

Comment:
oh I just remembered the other reason I didn't do this. I think if anything else is running in the same process space and has a handle to the engine then disposing it might break those other consumers.